src/utils/email-client/index.ts

```diff
@@ -0,0 +1,29 @@
+import Config from '../../config';
+import * as sgMail from '@sendgrid/mail';
+
+
+const defaults = {
+    from: 'Coding Coach <no-reply@mail.codingcoach.io>',
+};
+
+
+class EmailFacade {
```

Comment:
should we use a nestjs service here? those are suppose to be singleton, then we can use the dependency injection to send emails from controllers or other services.

Reply:
[Providers](https://docs.nestjs.com/providers)?

Reply:
yeah, seems like everything is a provider in nest:

> Many of the basic Nest classes may be treated as a provider - services, repositories, factories, helpers, and so on.

https://docs.nestjs.com/providers#services

Reply:
OK.  This has been transitioned over to a service.